lib/tasks/reporting.rake

```diff
@@ -133,6 +133,7 @@ namespace :reporting do
   end
 
   namespace :consultations do
+    desc "Report consultations opened after a specific date"
```

Comment:
I'd suggest this instead, which implies the specific date is included (currently reads as though the specific date is not included).

```suggestion
    desc "Report consultations opened since a specific date"
```